src/molecules/cardDate/cardDate.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import React from 'react';
+import { CardDate } from './cardDate';
+
+export default {
+  title: 'Molecules/CardDate',
+  component: CardDate,
+  argTypes: {},
+} as ComponentMeta<typeof CardDate>;
+
+const Template: ComponentStory<typeof CardDate> = ({ date, state }) => (
+  <CardDate date={date || 'Nov 31st, 2021'} state={state || 'Upcoming'} />
```

Comment:
@ArinCantCode  all defaultValues should be stored in obj, const defaultParams = {data: "Nov 31" }